integrations/gcp/gcp_core/utils.py

```diff
@@ -101,7 +101,13 @@ def get_service_account_project_id() -> str:
     "get project id associated with service account"
     try:
         default_credentials = json.loads(get_credentials_json())
-        project_id = default_credentials["quota_project_id"]
+        project_id = default_credentials.get("project_id_") or default_credentials.get(
+            "quota_project_id_"
+        )
+
+        if not project_id:
+            raise KeyError("project_id or quota_project_id")
```

Comment:
Change the phrasing here

Reply:
I meant to something amongst the lines of:
`Couldn't find keys project_id or quota_project_id in your credentials file.`

Reply:
okay, got you, the reason why its this way is because, line 122, add the prefix to the error string

Reply:
Eventually, this is what we get when a key error is raise

`Couldn't figure out the service account's project id. Key: project_id or quota_project_id doesn't exist in the credentials file."`

Reply:
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/964de3b9-d7a9-474c-a3d3-41f0d8855863">

Reply:
OHHH nice of nvm you can leave `project_id or quota_project_id`